### PR TITLE
Bind Delete shortcut with Delete Cluster, Topic command

### LINKED ISSUE #79: Bind Delete shortcut with Delete Cluster, Topic command
Bind Delete shortcut with Delete Cluster, Topic command

### PR DESCRIPTION
**Pull request by angelozerr**
Bind Delete shortcut with Delete Cluster, Topic command

Fixes #79

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR adds trashcan icon for topics

image

and gives the capability to use `Delete` key to execute delete command for topic or cluster. It works only if you select one item. I wanted to support multiple delete (which I think can be helpful for deleting several topics in same time) but as we can have some warn message according to the cluster configuration, I gave up this idea, but I think it should be nice to have multiple delete. 

@fbricon what do you think?

**Comment by fbricon** [comment]
I works great but it's not obvious the delete (cmd+backspace) key is supposed to work, from the context menu
Screenshot 2021-01-13 at 12 06 09

We basically have 3 commands doing the same thing: deleteSelected, deleteCluster, deleteTopic. We prolly want to keep the generic deleteSelected command (label: Delete) for everything, so the shortcut appears in the context menu like
Screenshot 2021-01-13 at 12 06 31

**Comment by angelozerr** [comment]
> . We prolly want to keep the generic deleteSelected command (label: Delete) for everything, so the shortcut appears in the context menu like

fixed

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to Kafka extension will be documented in this file.
 - Click on empty Kafka explorer to add a new cluster. See [#76](https//github.com/jlandersen/vscode-kafka/pull/76).
 - Added glob patterns to filter topics (`kafka.explorer.topics.filters`) and consumer groups (`kafka.explorer.consumers.filters`) out of the Kafka explorer. See [#74](https://github.com/jlandersen/vscode-kafka/pull/74).
 - Kafka Explorer item labels can now be copied to the clipboard (supports multi selection). See [#68](https://github.com/jlandersen/vscode-kafka/issues/68).
+- Bind Delete shortcut with Delete Cluster, Topic command. See [#79](https://github.com/jlandersen/vscode-kafka/issues/79)
```
Comment (fbricon) [inline CHANGELOG.md:9]: Selected Cluster or Topic can now be deleted via the `Delete` shortcut (`Cmd+Backspace` on Mac)

Comment (angelozerr) [reply CHANGELOG.md]: fixed

**Comment by fbricon** [comment]
Thanks @angelozerr!

**Pull request merged**